analyzers/src/SonarAnalyzer.Common/Helpers/WildcardPatternMatcher.cs

```diff
@@ -101,13 +96,13 @@ private static string ToRegexp(string wildcardPattern, string directorySeparator
                     {
                         // Single asterisk
                         // Zero or more characters excluding directory separator
```

Comment:
Can be merged to a single line. 
Also above with the double
```suggestion
                        // Single asterisk - Zero or more characters excluding directory separator
```